mozilla/fx-private-relay
# MPP-79: Add `ruff` Python linter

This PR adds the [ruff](https://docs.astral.sh/ruff/) tool as a Python linter. I've enabled a few rules, but there are [plenty more](https://docs.astral.sh/ruff/rules/) we could add if we find they would fix issues or flag issues from code review.

This PR fixes #162.

# How to test:

* Install with `pip install -r requirements`
* Run locally with `ruff check .`
* See some other errors by enabling rules like `ruff check --select RET .`
